fairmq/shmem/FairMQMessageSHM.h

```diff
@@ -9,6 +9,7 @@
 #define FAIRMQMESSAGESHM_H_
 
 #include <fairmq/shmem/Manager.h>
+#include "Common.h"
```

Comment:
Let's keep this include in the cxx file and fwd declare `MetaHeader` here

Reply:
it's an ungly namespaced one, but ok, will have to cleanup namespaces here anyway